packages/integration-tests/src/test-projects/data-storage-auth-with-triggers-ts/amplify/auth/resource.ts

```diff
@@ -1,5 +1,9 @@
 import { defineAuth, secret } from '@aws-amplify/backend';
-import { defaultNodeFunc } from '../function.js';
+import { defaultNodeFunc, funcWithSchedule } from '../function.js';
+
+const customEmailSenderFunction = {
+  handler: funcWithSchedule,
```

Comment:
I think we should use another function as `funcWithSchedule` has a cron job to invoke it every minute which may make testing this inaccurate.

Reply:
We would probably want a new/different function with same SQS setup for a different queue and no schedule set. That way only message in the queue will be from when you create a new user in `triggerEmailSending` and not from scheduled invocations.